x-pack/plugins/cloud_integrations/cloud_chat/server/config.ts

```diff
@@ -5,74 +5,29 @@
  * 2.0.
  */
 
-import { get, has } from 'lodash';
-import { schema, TypeOf } from '@kbn/config-schema';
+import { schema } from '@kbn/config-schema';
 import { PluginConfigDescriptor } from '@kbn/core/server';
 
-import { DEFAULT_TRIAL_BUFFER } from '../common/constants';
+const configSchema = schema.object({});
```

Comment:
I think that we need to keep `enabled` (even when flagged as _unused_ below) so that the plugin is disabled. Otherwise, it'll be enabled even for non-Cloud deployments.

```suggestion
const configSchema = schema.object({
  enabled: schema.boolean({ defaultValue: false }),
});
```

Reply:
nice, thank you!